css/style.css

```diff
@@ -16,4 +15,307 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/*styling header*/
+
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0px 50px;
+}
+
+.nav-bar {
+  text-align: right;
+  height: 70px;
+  line-height: 70px;
+}
+
+.menu {
+  margin: 0px 30px 0px 0px;
+}
+
+.menu a {
+  clear: right;
+  text-decoration: none;
+  color: gray;
+  margin: 0px 10px;
+  line-height: 35px;
+}
+
+.menu a:hover {
+  cursor: pointer;
+  color: rgb(238, 132, 2);
+}
+
+span {
+  font-weight: 500;
+}
+
+label {
+  margin: 0px 40px 0px 0px;
+  font-size: 26px;
+  line-height: 70px;
+  display: none;
+  width: 26px;
+  float: right;
+}
+
+#toggle {
+  display: none;
+}
+
+/*styling article*/
+
+.intro-karma {
+  background-image: url(/img/first-background.jpg);
```

Comment:
The path to the image is not correct, so it is not displaying. Your CSS file is in a subfolder, so you need to go up a level before going into the img folder

Reply:
Thanks for the hint. I've correct the files accessibility.